src/firebase/api.ts

```diff
@@ -10,12 +10,12 @@ const firebaseConfig = {
   messagingSenderId: '1047480032559',
   appId: '1:1047480032559:web:dc4c9d845104ba1f8c55da',
   measurementId: 'G-6WS3MMV659',
-};
+}
 
-firebase.initializeApp(firebaseConfig);
+firebase.initializeApp(firebaseConfig)
 
-const emailsIDCollection = firebase.firestore().collection('emails_id');
+const emailsIDCollection = firebase.firestore().collection('emails_id')
 
-const addEmail = (email: string): Promise<void> => emailsIDCollection.doc(email).set({ email });
+export const addEmail = (email: string): Promise<void> => emailsIDCollection.doc(email).set({ email })
 
-export default addEmail;
+export default firebase
```

Comment:
again why are we exporting `firebase`? It should be fine to just export the function `addEmail`

Reply:
Yes, for now we can export the addEmail alone, I was working on more features of firebase so I exported it but will remove it for now :)

Reply:
still not understanding why you would need to export firebase. You can import it with `import * as firebase from 'firebase/app'` there too no? I guess I'll see :)